docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+web:
+    build: .
+    volumes:
+        - .:/app
```

Comment:
@jderusse Wow, thanks for this detailed explanation! It's looks very useful for performance boost :+1: